Java/src/test/java/AirportTest.java

```diff
@@ -92,9 +92,9 @@ public void testHasAtLeastOneBomberInMilitaryPlanes() {
     @Test
     public void testExperimentalPlanesHasClassificationLevelHigherThanUnclassified(){
         Airport airport = new Airport(planes);
-        List<experimentalPlane> experimentalPlanes = airport.getExperimentalPlanes();
+        List<ExperimentalPlane> ExperimentalPlanes = airport.getExperimentalPlanes();
         boolean hasUnclassifiedPlanes = false;
-        for(experimentalPlane experimentalPlane : experimentalPlanes){
+        for(ExperimentalPlane experimentalPlane : ExperimentalPlanes){
```

Comment:
I'm commenting to the incorrect line (just to keep in mind), because comments can be left only to modified lines - please simplify some assertEquals to assertTrue and add error messages to all asserts. Also, please refactor testHasAtLeastOneBomberInMilitaryPlanes() method

Reply:
Assert was changed.
Alert messages were added to all tests.
From testHasAtLeastOneBomberInMilitaryPlanes() was refactored.

Reply:
will discuss on the q&a session

Reply:
Done